services/workflows-service/src/alert/alert.service.intg.test.ts

```diff
@@ -1631,12 +1632,242 @@ describe('AlertService', () => {
+                ...ALERT_DEFINITIONS.MGAV_CC.inlineRule,
+                options: {
+                  ...ALERT_DEFINITIONS.MGAV_CC.inlineRule.options,
+                  transactionFactor: 1,
+                },
+              },
+
+              enabled: true,
+            },
+            project,
+          ),
+        });
+
+        const counterPartyByType = (businessType: string) =>
+          createBusinessCounterparty({
+            prismaService,
+            projectId: project.id,
+            businessTypeFn: () => businessType,
+          });
+
+        counterpartiesA = await Promise.all(
+          new Array(3).fill(null).map(async () => counterPartyByType('businessA')),
+        );
+
+        await Promise.all(
+          new Array(3).fill(null).map(async (_, i) =>
+            counterpartiesA.map(async _counteryparty =>
+              new TransactionFactory({
+                prisma: prismaService,
+                projectId: project.id,
+              })
+                .withCounterpartyBeneficiary(_counteryparty.id)
+                .withEndUserOriginator()
+                .direction(TransactionDirection.inbound)
+                .paymentMethod(PaymentMethod.credit_card)
+                .transactionDate(faker.date.past(2))
+                .count(1 + i)
+                .create(),
+            ),
+          ),
+        );
+
+        await Promise.all(
+          new Array(2).fill(null).map(async (_, i) =>
+            counterpartiesA.splice(1, 2).map(async _counteryparty =>
+              new TransactionFactory({
+                prisma: prismaService,
+                projectId: project.id,
+              })
+                .withCounterpartyBeneficiary(_counteryparty.id)
+                .withEndUserOriginator()
+                .direction(TransactionDirection.inbound)
+                .paymentMethod(PaymentMethod.credit_card)
+                .transactionDate(faker.date.recent(2))
+                .count(i % 2 === 0 ? 3 : 2)
+                .create(),
+            ),
+          ),
+        );
+
+        await new TransactionFactory({
+          prisma: prismaService,
+          projectId: project.id,
+        })
+          // @ts-ignore
+          .withCounterpartyBeneficiary(counterpartiesA[0].id)
+          .withEndUserOriginator()
+          .direction(TransactionDirection.inbound)
+          .paymentMethod(PaymentMethod.credit_card)
+          .transactionDate(faker.date.recent(5))
+          .count(10)
+          .create();
+
+        await new TransactionFactory({
+          prisma: prismaService,
+          projectId: project.id,
+        })
+          // @ts-ignore
+          .withCounterpartyBeneficiary(counterpartiesA[0].id)
+          .withEndUserOriginator()
+          .direction(TransactionDirection.inbound)
+          .paymentMethod(PaymentMethod.credit_card)
+          .transactionDate(faker.date.past(5))
+          .count(10)
+          .create();
+      });
+
+      it(`Trigger an alert`, async () => {
```

Comment:
Maybe write a test when it doesn't trigger an alert?

Reply:
i talked about it with Alon, we'll do once we will get their requirements for this rule because we assume it will be change